### PR TITLE
Fix Tracker Alignment all-in-one tool for py3 (again)

### LINKED ISSUE #29337: Python 3 failure in TkAlAllInOneTool unit test
It appears that #29255 introduced a failure in TkAlAllInOneTool unit test python3 IBs
```
TESTING all-in-one tool ...
Requesting file information for '/MinimumBias/Run2017A-TkAlMinBias-PromptReco-v1/ALCARECO' from DAS... Done.
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_PY3_X_2020-03-29-2300/bin/slc7_amd64_gcc820/validateAlignments.py", line 762, in &lt;module&gt;
    main()
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_PY3_X_2020-03-29-2300/bin/slc7_amd64_gcc820/validateAlignments.py", line 735, in main
    jobs.extend( ValidationJob(validation, config, options, 1) )
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_PY3_X_2020-03-29-2300/bin/slc7_amd64_gcc820/validateAlignments.py", line 132, in __init__
    self.validation = self.__getValidation( self.valType, self.valName,
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_PY3_X_2020-03-29-2300/bin/slc7_amd64_gcc820/validateAlignments.py", line 173, in __getValidation
    Alignment( alignments.strip(), config ), config )
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_PY3_X_2020-03-29-2300/python/Alignment/OfflineValidation/TkAlAllInOneTool/alignment.py", line 57, in __init__
    self.conditions = self.__getConditions( config, section )
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_PY3_X_2020-03-29-2300/python/Alignment/OfflineValidation/TkAlAllInOneTool/alignment.py", line 252, in __getConditions
    self.__testDbExist(condition["connectString"], condition["tagName"])
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_PY3_X_2020-03-29-2300/python/Alignment/OfflineValidation/TkAlAllInOneTool/alignment.py", line 260, in __testDbExist
    elif "\n"+tagname not in conddb("--db", dbpath.split("sqlite_file:")[1], "listTags"):
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_PY3_X_2020-03-29-2300/python/Alignment/OfflineValidation/TkAlAllInOneTool/helperFunctions.py", line 224, in conddb
    with NamedTemporaryFile(bufsize=0) as sys.stdout:
TypeError: NamedTemporaryFile() got an unexpected keyword argument 'bufsize'
```

### PR DESCRIPTION
**Pull request by mmusich**
resolves https://github.com/cms-sw/cmssw/issues/29337

#### PR description:

Title says it all. Removed contraption to list tags in favor of DB group supported `conddbllib` commands 

#### PR validation:

Unit test has been exercised successfully with this PR in `CMSSW_11_1_PY3_X_2020-03-29-2300`

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Not a backport, and no backport is meant.

### CODE REVIEW

**Comment by makortel** [comment]
Thanks!

**Comment by makortel** [comment]
@cmsbuild, please test

**Comment by mmusich** [comment]
wait... I missed a commit!

**Comment by makortel** [comment]
@cmsbuild, please abort

**Comment by mmusich** [comment]
OK, now is OK for testing.
@makortel is there a way to instruct the bot to run under a python3 build?

**Comment by makortel** [comment]
test parameters
  - release = CMSSW_11_1_PY3_X

**Comment by makortel** [comment]
@cmsbuild, please test

> is there a way to instruct the bot to run under a python3 build?

Yes, trying now.

**Comment by makortel** [comment]
@cmsbuild, please test for CMSSW_11_1_PY3_X

Let me try again to trigger py3 tests.

**Comment by mmusich** [comment]
> DQMHistoTests: Total failures: 723

These failures must come from something else. The update is here is not affecting anything run in relvals.

**Comment by davidlange6** [comment]
Right - Py3 and py2 won’t give the same results.


On Mar 31, 2020, at 1:51 PM, Marco Musich <notifications@github.com<mailto:notifications@github.com>> wrote:



DQMHistoTests: Total failures: 723

These failures must come from something else. The update is here is not affecting anything run in relvals.

**Comment by silviodonato** [comment]
kind reminder @pohsun @tlampen @tocheng @christopheralanwest

**Comment by pohsun** [comment]
+1

**Comment by silviodonato** [comment]
+1
